docs/source/conda-package-walkthrough/package_anatomy.rst

```diff
@@ -0,0 +1,123 @@
+********************
+Anatomy of a package
+********************
+
+What is a “conda package”?
+--------------------------
+
+* A compressed tarball file that may contain
+
+  * python or other modules
+
+  * executables
+
+  * etc.
+
+* Package format is the same across platforms
+
+* Conda understands how to install these packages into a conda environment
+  so that it may be available when that environment is active
+
+Using packages
+---------------
+
+* You may search for packages
+
+.. code-block:: bash
+
+  conda search scipy
+
+
+* You may install a package
+
+.. code-block:: bash
+
+  conda install scipy
+
+
+* You may build a package
+
+.. code-block:: bash
+
+  conda build my_fun_package
+
+Package structure
+-----------------
+
+.. code-block:: bash
+
+  .
+  ├── bin
+  │   └── pyflakes
+  ├── info
+  │   ├── LICENSE.txt
+  │   ├── files
+  │   ├── index.json
+  │   ├── paths.json
+  │   └── recipe
+  └── lib
+    └── python3.5
+
+* bin contains relevant binaries for the package
+
+* lib contains the relevant library files (eg. the .py files)
+
+* info contains package metadata
+
+Info
+----
+
+* files
+
+  * a list of all the files in the package (not included in info/)
+
+* index.json
+
+  * metadata about the package including platform, version, dependencies, build info
+
+* paths.json
+
+  * a list of files in the package, along with their associated SHA-256, size in bytes,
+    and the type of path (eg. hardlink vs. softlink)
+
+index.json
```

Comment:
I think this can go under the index.json bullet point. Initially I put it here because it worked better for slides